src/ElectroGrammar.g4

```diff
@@ -3,18 +3,10 @@
  */
 
 grammar ElectroGrammar;
-import Alphabet;
+import PackageSize, Units;
 
-capacitance : NUMBER CPREFIX FARAD?;
+electro_grammar : capacitor;
```

Comment:
This is fine for now but ultimately I want the parser to work on individual specs like `50V` and `10%` without outputting a complete component.

Reply:
To be clear: It's fine to just port the existing functionality for now. Just something to keep in mind for the future.

Reply:
Yes, that makes sense